MedallionShell.Tests/CommandLineSyntaxTest.cs

```diff
@@ -14,7 +14,7 @@ public class CommandLineSyntaxTest
         [Test]
         public void TestArgumentValidation([Values] bool isWindowsSyntax)
         {
-            var syntax = isWindowsSyntax ? new WindowsCommandLineSyntax() : new MonoUnixCommandLineSyntax().As<CommandLineSyntax>();
+            var syntax = new CrossPlatformCommandLineSyntax().As<CommandLineSyntax>();
```

Comment:
I renamed `WindowsCommandLineSyntax` to `CrossPlatformCommandLineSyntax` but didn't delete it (even though it's the only class that extends `CommandLineSyntax`). Let me know if you want to delete it.

Reply:
Pretty sure this class is _not_ cross platform, though (or maybe it is for the specific purposes of .NET).

That said, this class is public surface area so removing/renaming would be a breaking change.